docs/gettingstarted/ansible-docker-podman.md

```diff
@@ -1,47 +1,43 @@
-## Notice
-SC4S installation can now be automated with Ansible. All you need to do now is provide a host on which you want to run SC4S and basic configuration (Splunk endpoint, HEC token, TLS configuration, etc.).
+SC4S installation can be automated with Ansible. To do this, you provide a list of hosts on which you want to run SC4S and basic configuration information, such as the Splunk endpoint, HEC token, and TLS configuration.
 
-# Initial Configuration
+# Step 1: Prepare your initial configuration
 
-All you need to do before running sc4s with Ansible is providing `env_file`. In the env file provide at least proper Splunk endpoint and HEC token.
-Create a file in `ansible/resources` catalog or edit [example file](/ansible/resources/env_file).
+1. Before running SC4S with Ansible, provide `env_file` with your Splunk endpoint and HEC token. To do this, create the file in `ansible/resources` catalog or edit the [example file](/ansible/resources/env_file):
 
 ``` dotenv
 --8<---- "ansible/resources/env_file"
 ```
-Next provide a host on which you want to run Docker Swarm cluster and host application in inventory file:
+2. Provide a list of hosts on which you want to run your Docker Swarm cluster and the host application in the inventory file:
 ``` yaml
 --8<---- "ansible/inventory/inventory.yaml"
 ```
-## Deploy SC4S
-Now you can run ansible playbook to deploy the application if you have ansible installed on your host
-or use docker ansible image provided in the package:
+# Step 2: Deploy SC4S on your configuration
+1. If you have Ansible installed on your host, run the Ansible playbook to deploy SC4S. Otherwise, use the Docker Ansible image provided in the package:
 ```bash
 # From repository root
 docker-compose -f ansible/docker-compose.yml build
 docker-compose -f ansible/docker-compose.yml up -d
 docker exec -it ansible_sc4s /bin/bash
 ```
-Once you are in containers remote shell you can run Docker Swam ansible playbook.
-If you are authenticating via username/password:
+2. If you used the Docker Ansible image in step 1, then from your container remote shell, authenticate to and run the Docker Swam Ansible playbook.
```

Comment:
Name collision, step 1 was used in the header of the previous section

Reply:
good point, that is totally confusing! I changed it to "previous step" but I'm making notes and maybe we can rethink the numbering and structure oin the next round of reviews.